File: BackEndAPI/Data/Entites/PaymentToCreditor.cs
using BackEndAPI.Models;
using System;
using System.Collections.Generic;

namespace BackEndAPI.Data.Entites
{
    public class PaymentToCreditor
    {
        public int CaseId { get; set; }
        public Case Case { get; set; }
        public int CharityId { get; set; }
        public Charity Charity { get; set; }
        public int CreditorId { get; set; }
        public Creditor Creditor { get; set; }
        public decimal Paid_Amount { get; set; }
      
        public DateTime Time { get; set; }

        public static explicit operator PaymentToCreditor(List<PaymentToCreditor> v)
        {
            throw new NotImplementedException();
        }

    }
}

File: FrontEnd/src/app/Models/PaymentToCreditor.ts
import { Case } from "./Case"
import { Charity } from "./Charity"
import { Creditor } from "./Creditor"

export interface PaymentToCreditor {
  caseId: number
  case: Case
  charityId: number
  charity: Charity
  creditorId: number
  creditor: Creditor
  paid_Amount: number
  time: string
}
